src/main/java/training/supportbank/models/Transaction.java

```diff
@@ -0,0 +1,44 @@
+package training.supportbank.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public class Transaction {
+    private final LocalDate date;
+    @SerializedName("fromAccount")
+    private final String from;
+    @SerializedName("toAccount")
+    private final String to;
+    private final String narrative;
+    private final BigDecimal amount;
+
+    public LocalDate getDate() {
+        return date;
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public String getNarrative() {
+        return narrative;
+    }
+
+    public BigDecimal getAmount() {
+        return amount;
+    }
+
+    public Transaction(LocalDate date, String from, String to, String narrative, BigDecimal amount) {
```

Comment:
Really minor point: in Java it's conventional to put the constructor before any of the other methods.